### ISSUE TITLE
Maintain spacing when using "Correct Usings"

### ISSUE THREAD

**Issue by jdpjamesp** [omecra/de.omecra.oedt.updatesite#103]
I like having my USING statements in alphabetical order, and in tabular format for readaibility. Unfortunately Correct Usings fixes the order of the USING statements, but at the expense of the spacing. Please can you add the option to maintain spacings. 

```
USING Consultingwerk.Framework.*                FROM PROPATH.
USING Beckelmann.Angebot.BL.*                   FROM PROPATH.
USING Beckelmann.Auftrag.BL.*                   FROM PROPATH.
USING Beckelmann.SonderpreistypAufteilung.BL.*  FROM PROPATH.
USING Consultingwerk.*                          FROM PROPATH.
USING Consultingwerk.Assertion.*                FROM PROPATH.
USING Consultingwerk.OERA.*                     FROM PROPATH.
USING Consultingwerk.OERA.Exceptions.*          FROM PROPATH.
USING Consultingwerk.SmartFramework.*           FROM PROPATH.
USING Consultingwerk.Util.*                     FROM PROPATH.
USING Progress.Lang.*                           FROM PROPATH.
```

Becomes

```USING Beckelmann.Angebot.BL.* FROM PROPATH.
USING Beckelmann.Auftrag.BL.* FROM PROPATH.
USING Beckelmann.SonderpreistypAufteilung.BL.* FROM PROPATH.
USING Consultingwerk.* FROM PROPATH.
USING Consultingwerk.Assertion.* FROM PROPATH.
USING Consultingwerk.Framework.* FROM PROPATH.
USING Consultingwerk.OERA.* FROM PROPATH.
USING Consultingwerk.OERA.Exceptions.* FROM PROPATH.
USING Consultingwerk.SmartFramework.* FROM PROPATH.
USING Consultingwerk.Util.* FROM PROPATH.
USING Progress.Lang.* FROM PROPATH.
```

**Comment by Ton-Pe** [comment]
+1 vote

**Comment by LievenDeFoor** [comment]
I really can't understand the fascination of aligning code like this. It's something I've only seen with some ABL developers.
I have yet to encounter such coding style in other languages...
This means you constantly have to reformat existing code when adding a new item.
That can't be a pleasant diff when reviewing or merging code...

**Comment by mikefechner** [comment]
This ticket here is not meant to be a discussion on coding style. But I bet most other languages you refer to, don't have a FROM .... option on USING statements following the referenced packages. And if not aligned, it just takes more effort, to view the actual packages in the middle of the statement. So in this particular language (that OEDT is an editor for) it provides added value. 

> That can't be a pleasant diff when reviewing or merging code...

Maybe you need to look at better diff/merge tools than those that come with certain SCM tools.

**Comment by LievenDeFoor** [comment]
Don't be offended Mike, I'm not looking for trouble.
Consider it a downvote to this feature request.
I'm using p4merge by the way.
Cheers

**Comment by lneumeier** [comment]
Fixed in OEDT 9.3.5.
